### PR TITLE
Upgrade: README 👓

### PR DESCRIPTION
**Pull request by SharpMox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,76 +1,84 @@
...
+  - [Cache](#package--cache)
+  - [Server-side rendering](#postbox--server-side-rendering)
+  - [Blacklist](#warning--blacklist)
+- [Feedbacks](#baby_bottle--feedback)
 
-## :thinking: Presentation
+---
 
-This library for React has a very simple goal: prevent the use of **dagerouslySetInnerHTML** function.
+## :thinking:  Why ?
```
Comment (baptooo) [inline README.md]: Careful there is two spaces between "thinking" and "why", is it intentional ?

File: README.md [hunk 2]
```diff
@@ -1,76 +1,84 @@
...
+And React actually [warns you about it](https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml).
+
+
+### :angel: **`safelySetInnerHTML`** :angel: 
+
+Is a tool to solve this issue by filtering and creating automatic the react dom and return it to your component as children elements.
 
-By default, only few tags are allowed so you don't need to sanitize the HTML string yourself, just configure
-the scope for your needs.
+By default, only `<strong>` and `<a>` tags  and only the `href` attribute are allowed.
```
Comment (baptooo) [inline README.md:45]: There is the default config following this sentence, it is a bit redundant, isn't it ?

File: README.md [hunk 3]
```diff
@@ -1,76 +1,84 @@
...
   ALLOWED_TAGS: [
     'strong',
-    'a'
+    'a',
   ],
   ALLOWED_ATTRIBUTES: [
-    'href'
+    'href',
   ],
   KEY_NAME: 'ssih-tag-'
```
Comment (baptooo) [inline README.md]: missing the `,` also here

File: README.md [hunk 4]
```diff
@@ -1,76 +1,84 @@
...
 }
 ```
+You will of course be able [extend this config to the scope of your needs](#wrench-configuration).
```
Comment (baptooo) [inline README.md]: This is not extend, but override maybe just like this ?

>You will of course be able [modify this configuration to the scope of your needs](#wrench-configuration).

File: README.md [hunk 5]
```diff
@@ -1,76 +1,84 @@
...
+Simply install the library with npm:
 
 ```sh
 $ npm install -D safely-set-inner-html
 ```
 
-## :electric_plug: Usage
+---
+
+## :zap:  Usage
```
Comment (baptooo) [inline README.md]: Why remove the 🔌 ? I like it :'(

File: README.md [hunk 6]
```diff
@@ -144,18 +117,20 @@ import SafelySetInnerHTML from 'safely-set-inner-html';
...
     'a',
-    'strong'
+    'strong',
+    ...
   ],
   ALLOWED_ATTRIBUTES: [
     'href',
-    'class'
+    'class',
+    ...
```
Comment (baptooo) [inline README.md]: Be careful this is not a valid JS snippet with this, with a comment it will be better

File: README.md [hunk 7]
```diff
@@ -144,18 +117,20 @@ import SafelySetInnerHTML from 'safely-set-inner-html';
 const mySafelySetInnerHTML = new SafelySetInnerHTML({
   ALLOWED_TAGS: [
     'a',
-    'strong'
+    'strong',
+    ...
```
Comment (baptooo) [inline README.md]: Be careful this is not a valid JS snippet with this, with a comment it will be better

File: README.md (hunk 2 above)
Comment (SharpMox) [reply README.md:45]: It is, but just showing a code snippet without a support text seems poor to me.

File: README.md (hunk 1 above)
Comment (SharpMox) [reply README.md]: Yes, i've done this on every title.
To have the link pattern look like : `[TitleName](#emoji_name--titleName)`

File: README.md (hunk 3 above)
Comment (SharpMox) [reply README.md]: 🙈

File: README.md (hunk 4 above)
Comment (SharpMox) [reply README.md]: Ha yes perfect ! Thanks :)

File: README.md (hunk 5 above)
Comment (SharpMox) [reply README.md]: 😅

File: README.md (hunk 7 above)
Comment (SharpMox) [reply README.md]: 👌

File: README.md (hunk 6 above)
Comment (SharpMox) [reply README.md]: 👌

File: README.md (hunk 1 above)
Comment (baptooo) [reply README.md]: But why ? now there is two spaces between the emoji and the title correct ? 🤔

File: README.md (hunk 2 above)
Comment (baptooo) [reply README.md:45]: Ok why not.
Anyway, I am planning to enhance the default config to have a larger scope of tags / attributes so there will be a dedicated section for this I think.

File: README.md [hunk 8]
```diff
@@ -1,76 +1,84 @@
...
-- [Presentation](#thinking-presentation)
-- [Getting started](#inbox_tray-getting-started)
-- [Usage](#electric_plug-usage)
-  - [Cache](#cache)
-  - [Server-side rendering](#server-side-rendering)
-  - [Blacklist warnings](#blacklist-warnings)
-- [Live demo](#joystick-live-demo)
-- [Configuration](#wrench-configuration)
-- [Feedbacks](#feedbacks)
+- [Introduction](#thinking--why-)
```
Comment (baptooo) [inline README.md]: Introduction wording goes to "why", it is disturbing

File: README.md (hunk 1 above)
Comment (SharpMox) [reply README.md]: ¯\_(ツ)_/¯ I'll switch it back to single space

File: README.md (hunk 2 above)
Comment (SharpMox) [reply README.md:45]: Perfect 👌

**Pull request closed without merging**